### PR TITLE
Added Cart and wishlist

### PR DESCRIPTION
**Pull request by Anurag021**
Added Code for Cart and wishlist
### Added pages
- cart.css
- wishlist.css
- cart.html
- wishlist.html

### Preview Link
Cart Page : https://hardcore-sinoussi-9e4133.netlify.app/cart-and-wishlist/cart
WishList page: https://hardcore-sinoussi-9e4133.netlify.app/cart-and-wishlist/wishlist

### CODE REVIEW

**Comment by kgajriaAgarwal** [comment]
image
footer should be fixed on to the bottom of the screen

**Review by Vishnu-Aithal — APPROVED** [review]
Looks Good to me !

**Review by Maianki — APPROVED** [review]
Great work Anurag. Loved your code well commented and good to see that you have made HTML semantic.

File: Cart-And-Wishlist/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+                        <span>Rs. 249 </span>
+                        <strike>Rs. 500</strike>
+                        <i class="fa-solid fa-heart wishlist-label"></i>
+                    </p> 
+                    <div class="vertical-card-button">
+                        <button class="whishlist-btn">Move To Cart</button>
+                        <button class="whishlist-btn"> Remove From WishList</button>    
+                    </div>
+                </div>
+                <span class="card-close">❌</span>
```
Comment (rahulk31) [inline Cart-And-Wishlist/wishlist.html]: I see you have "remove from wishlist" button as well as the cross sign on the card. Both will serve the same purpose of removing the product from the wishlisted page, so my suggestion would be to keep only one(any) acc. to your personal preference.

**Review by rahulk31 — APPROVED** [review]
UI suggestion:
image

To give some padding top to the footer and also bit of margin-top.

Overall code looks well structured and good use of class names. You can merge it 👍

File: Cart-And-Wishlist/wishlist.html (hunk 1 above)
Comment (Anurag021) [reply Cart-And-Wishlist/wishlist.html]: Thanks for review, Removed the card-close one and implemented below comment too.

**Pull request merged**